apps/api/src/secret/secret.types.ts

```diff
@@ -16,4 +16,10 @@ export interface SecretWithEnvironment extends Secret {
   environment: Environment
 }
 
+export type SecretsByEnvironment = {
```

Comment:
Remove these newly added type defs because,

1. The existing ones are interfaces and we want to maintain the similarity
2. These are only used in one place which can be automatically type inferred.